ui/src/containers/DashboardSharing/index.js

```diff
@@ -255,8 +279,28 @@ const ModelFormComponent = ({
     </div>}
 
 
+    {(filterMode === JWT_SECURED) &&
+    <div className="form-group" style={{ marginBottom: 25 }}>
+      <label htmlFor={filterJwtSecretId} >
+        Filter required
+      </label>
+      <span className={classNames('help-block', styles.contextHelp)}>Must a filter be passed to the URL for the dashboard to load data?</span>
+      <RadioGroup
+        id={filterRequiredId}
+        name="filterRequired"
+        value={filterRequired ? '1' : '0'}
```

Comment:
Seems like this wanted a string? So I'm casting a boolean to a string here `"1`/"0"` and then casting back to a boolean via in the onChange method